### PR TITLE
doobie metrics

### PR DESCRIPTION
**Pull request by iodkowski**

### CODE REVIEW

File: modules/doobie/src/main/scala/com/evolutiongaming/smetrics/DbMetrics.scala [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+import cats.implicits._
+import com.evolutiongaming.smetrics.MetricsHelper._
+
+import scala.concurrent.duration.FiniteDuration
+
+trait DbMetrics[F[_]] {
+  def query(time: FiniteDuration, success: Boolean): F[Unit]
+}
+
+object DbMetrics {
```
Comment (t3hnar) [inline modules/doobie/src/main/scala/com/evolutiongaming/smetrics/DbMetrics.scala:14]: as you put that into root `com.evolutiongaming.smetrics` package, I'd call it DoobieMetrics or TransactorMetrics, and avoid "generic" names

File: modules/doobie/src/main/scala/com/evolutiongaming/smetrics/TransactorMetered.scala [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+import cats.implicits._
+import com.evolutiongaming.catshelper.BracketThrowable
+import doobie.free.connection.ConnectionOp
+import doobie.util.transactor.{Interpreter, Transactor}
+
+object TransactorMetered {
+
+  def apply[F[_]: BracketThrowable: MeasureDuration](
+    transactor: Transactor[F],
+    dbMetrics: DbMetrics[F]
```
Comment (t3hnar) [inline modules/doobie/src/main/scala/com/evolutiongaming/smetrics/TransactorMetered.scala:15]: I think it makes sense to call it `metrics` in this scope

File: modules/doobie/src/main/scala/com/evolutiongaming/smetrics/DbMetrics.scala (hunk 1 above)
Comment (iodkowski) [reply modules/doobie/src/main/scala/com/evolutiongaming/smetrics/DbMetrics.scala:14]: but this trait can be used not only for doobie, maybe we can put it to `db` package?

Comment (t3hnar) [reply modules/doobie/src/main/scala/com/evolutiongaming/smetrics/DbMetrics.scala:14]: resting such simple thing will complicate adding new "doobie" related features if any. Not sure that worths doing that

Comment (iodkowski) [reply modules/doobie/src/main/scala/com/evolutiongaming/smetrics/DbMetrics.scala]: ok, fixed

File: modules/doobie/src/main/scala/com/evolutiongaming/smetrics/TransactorMetered.scala (hunk 2 above)
Comment (iodkowski) [reply modules/doobie/src/main/scala/com/evolutiongaming/smetrics/TransactorMetered.scala]: fixed

**Pull request merged**